_adr/76/index.adoc

```diff
@@ -0,0 +1,96 @@
+---
+num: 76
+title: "Handling usage of dynamically-async-created DNS records from dependent services"
+status: "Accepted"
+authors:
+  - "Gerard Ryan"
+tags:
+  - kafka
+applies_padrs: # What PADRs does this ADR apply?
+applies_patterns: # What APs does this ADR apply?
+---
+
+:link-adr-57: link:../57/index.html[ADR-57]
+:abbr-rhosak: pass:[<abbr title="Red Hat OpenShift Streams for Apache Kafka">RHOSAK</abbr>]
```

Comment:
I haven't been able to build the site on Fedora (it doesn't have a Ruby version old enough for this version of Jekyll) to see how well these attributes work when rendered. Managed to build on [ubi8/ruby-27](https://catalog.redhat.com/software/containers/ubi8/ruby-27/5f521b90acb94fdd904e98d7), but couldn't run (I gave up after troubleshooting past the first 2 or 3 issues).

Reply:
Created issue #47 for the issues I was having with building on Fedora.